packages/storybook/stories/BetaBanners/StoryBookBanner.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Link, mergeStyles, Stack, Text } from '@fluentui/react';
+import React from 'react';
+import { MICROSOFT_AZURE_PREVIEWS_URL } from '../constants';
+import { BannerPalette, blueBannerPalette } from './BannerPalettes';
+import { ImportantHeading } from './ImportantBannerHeading';
+
+/**
+ * @private
+ */
+export interface StoryBookBannerProps {
+  palette?: BannerPalette;
+  children?: JSX.Element;
+}
+
+/** @private */
+export const StoryBookBanner = (props: StoryBookBannerProps): JSX.Element => {
+  const palette: BannerPalette = props.palette ?? blueBannerPalette;
+  return (
+    <Stack className={bannerContainerStyles(palette.background)}>
+      <Stack.Item>
+        {props.children && (
+          <Stack style={{ display: 'inline-block' }}>
+            <ImportantHeading color={palette.header} />
+            {' - '}
+            {props.children}
+          </Stack>
+        )}
+        {!props.children && (
```

Comment:
This shouldn't pivot off prop.children, but be it's own component that uses StorybookBanner, i.e. something like
```
const SingleLineBetaBanner () => (
  <StoryBookBanner>
    <>
      <Text>This feature is currently in public preview and not recommended for production use. </Text>
      <Link  ... >
    </>
  </StorybookBanner>
);
```

Reply:
ah okay I see, will make this change!